roles/nodejs_scl/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+- name: 'Install SCL repository'
+  yum:
+    name: centos-release-scl-rh
+    state: present
+
+- name: 'Install Nodejs SCL'
+  yum:
+    name:
+      - "{{ nodejs_scl_version }}"
+      - "{{ nodejs_scl_version }}"
+    state: present
+
+- name: 'Enable SCL at login'
+  blockinfile:
+    dest: /etc/profile.d/enable-{{ nodejs_scl_version }}.sh
```

Comment:
This makes me wonder about the Ruby SCL. If we change the value, it continues to load the old SCL. Perhaps this needs to be static and just `enable-nodejs-scl.sh`. Same with Ruby.